scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,276 @@
+# Install sbt user qa
+RUN curl -L -o /home/qa/sbt.zip https://github.com/sbt/sbt/releases/download/v1.10.2/sbt-1.10.2.zip \
+  && unzip /home/qa/sbt.zip -d /home/qa \
+  && rm /home/qa/sbt.zip
+
+# Add tools like aws and sbt to the PATH
+ENV PATH /home/qa/.local/bin:/home/qa/sbt/bin:/home/qa/bin:${PATH}
+
+# Create symlink for sbt
+RUN ln -s /home/qa/sbt/bin/sbt /usr/local/bin/sbt
+
+RUN sudo chown -R $(whoami) /tmp/.sbt
```

Hadolint

[warning] Quote this to prevent word splitting.

(SC2046)